shared/components/ErrorPage500/index.scss

```diff
@@ -0,0 +1,22 @@
+.ErrorPage500 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  padding: 20px 0px;
+  &__text {
+    font-weight: normal;
+    text-align: center;
+  }
+  &__image {
+    margin: 20px 0px;
+  }
+}
+@media (min-width: $mobile-width) {
+  .ErrorPage500 {
+    &__text {
+      font-size: 24px;
```

Comment:
you could move this media query to be inline after line 9? like so:

```
&__text {
  font-weight: normal;
  text-align: center;
  @media (min-width: $mobile-width) {
    font-size: 24px;
  }
}
```
this isn't a pattern we really use so might be a bit confusing
could reduce line 16 and 17 down to be `.ErrorPage500__text`